### PR TITLE
fix: include default workerpool in list

### PR DESCRIPTION
**Pull request by kierramarie**
### Description

 The default worker pool was not originally included in the "workerpools" output variable. It has been updated to include it now.

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

fixes https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/issues/367

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by jor2 — DISMISSED** [review]

**Comment by jor2** [comment]
https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/issues/367

File: outputs.tf [hunk 1]
```diff
@@ -22,7 +22,7 @@ output "cluster_crn" {
 
 output "workerpools" {
   description = "Worker pools created"
-  value       = var.ignore_worker_pool_size_changes ? ibm_container_vpc_worker_pool.autoscaling_pool : ibm_container_vpc_worker_pool.pool
+  value       = merge(data.ibm_container_vpc_worker_pool.default_pool, var.ignore_worker_pool_size_changes ? ibm_container_vpc_worker_pool.autoscaling_pool : ibm_container_vpc_worker_pool.pool)
```
Comment (toddgiguere) [inline outputs.tf:25]: Spoke with author, we are going to try to do this with a single data block representing all pools to ensure that the final object blocks will always match.

**Review by toddgiguere — CHANGES_REQUESTED** [review]

**Review by toddgiguere — APPROVED** [review]

**Pull request merged**